js/app/modules/card/list.js

```diff
@@ -46,6 +47,13 @@ const List = new Module.Class({
         if (this.navigation_context)
             this.set_label_or_hide(this._navigation_context_label, this.navigation_context);
         this.set_thumbnail_frame_from_model(this._thumbnail_frame);
+
+        // FIXME: It's not clear what should go in the context label if model
+        // is not a video. This may need to be changed later on once we clarify
+        // that with other presets.
+        if (this.model instanceof Eknc.VideoObjectModel) {
```

Comment:
Why don't we just check for the duration property? I mean, instead of making it specific for Video, I think we plan to add audio as well, so the more flexible the better.